packages/babel-preset-react-app/index.js

```diff
@@ -84,10 +84,13 @@ if (env === 'test') {
   module.exports = {
     presets: [
       // Latest stable ECMAScript features
-      [require.resolve('babel-preset-latest'), {
-        'es2015': {
-          modules: false
-        }
+      [require.resolve('babel-preset-env'), {
+        targets: {
+          // We support everything React supports
+          'ie': 9
```

Comment:
I actually checked the plugins by hand, and there is a subtle difference.

Latest includes `transform-es2015-duplicate-keys`, Env does not.
Env includes `babel-plugin-check-es2015-constants`, latest does not.

This does result in slightly different code:

In:
```js
var obj = {
  a: 1,
  a: 5
}
```

env:
```
"use strict";

var obj = {
  a: 1,
  a: 5
};
```

latest:
```js
"use strict";

function _defineProperty(obj, key, value) { if (key in obj) { Object.defineProperty(obj, key, { value: value, enumerable: true, configurable: true, writable: true }); } else { obj[key] = value; } return obj; }

var obj = _defineProperty({
  a: 1
}, "a", 5);
```

I'm not sure which is proper behavior, but I assume there was a reason for this.
/cc @hzoo @existentialism